exchanges/deribit/deribit_wrapper.go

```diff
@@ -0,0 +1,1360 @@
+				continue
+			}
+			var cp currency.Pair
+			cp, err = currency.NewPairFromString(marketSummary[i].InstrumentName)
+			if err != nil {
+				return nil, err
+			}
+			var ct futures.ContractType
+			switch marketSummary[i].SettlementPeriod {
+			case "day":
+				ct = futures.Daily
+			case "week":
+				ct = futures.Weekly
+			case "month":
+				ct = futures.Monthly
+			case "perpetual":
+				ct = futures.Perpetual
+			}
+			var contractSettlementType futures.ContractSettlementType
+			if marketSummary[i].InstrumentType == "reversed" {
+				contractSettlementType = futures.Inverse
+			} else {
+				contractSettlementType = futures.Linear
+			}
+			resp = append(resp, futures.Contract{
+				Exchange:             d.Name,
+				Name:                 cp,
+				Underlying:           currency.NewPair(currency.NewCode(marketSummary[i].BaseCurrency), currency.NewCode(marketSummary[i].QuoteCurrency)),
+				Asset:                item,
+				SettlementCurrencies: []currency.Code{currency.NewCode(marketSummary[i].SettlementCurrency)},
+				StartDate:            marketSummary[i].CreationTimestamp.Time(),
+				EndDate:              marketSummary[i].ExpirationTimestamp.Time(),
+				SettlementType:       contractSettlementType,
+				IsActive:             marketSummary[i].IsActive,
+				MaxLeverage:          marketSummary[i].MaxLeverage,
+				Type:                 ct,
+			})
+		}
+	}
+	return resp, nil
+}
+
+// GetLatestFundingRates returns the latest funding rates data
+func (d *Deribit) GetLatestFundingRates(ctx context.Context, r *fundingrate.LatestRateRequest) ([]fundingrate.LatestRateResponse, error) {
+	if r == nil {
+		return nil, fmt.Errorf("%w LatestRateRequest", common.ErrNilPointer)
+	}
+	if !d.SupportsAsset(r.Asset) {
+		return nil, fmt.Errorf("%s %w", r.Asset, asset.ErrNotSupported)
+	}
+	var err error
+	available, err := d.GetAvailablePairs(r.Asset)
+	if err != nil {
+		return nil, err
+	}
+	if !available.Contains(r.Pair, true) || r.Pair.Quote.String() != "PERPETUAL" {
+		return nil, fmt.Errorf("%w pair: %v", futures.ErrNotPerpetualFuture, r.Pair)
+	}
+	r.Pair, err = d.FormatExchangeCurrency(r.Pair, r.Asset)
+	if err != nil {
+		return nil, err
+	}
+	var fri []FundingRateHistory
+	fri, err = d.GetFundingRateHistory(ctx, r.Pair.String(), time.Now().Add(-time.Hour*16), time.Now())
+	if err != nil {
+		return nil, err
+	}
+
+	resp := make([]fundingrate.LatestRateResponse, 0, len(fri))
+	for i := range fri {
+		rate := fundingrate.LatestRateResponse{
+			TimeChecked: time.Now(),
+			Exchange:    d.Name,
+			Asset:       r.Asset,
+			Pair:        r.Pair,
+			LatestRate: fundingrate.Rate{
+				Time: fri[i].Timestamp.Time(),
+				Rate: decimal.NewFromFloat(fri[i].Interest8H),
+			},
+		}
+		resp = append(resp, rate)
+	}
+	if len(resp) == 0 {
+		return nil, fmt.Errorf("%w %v %v", futures.ErrNotPerpetualFuture, r.Asset, r.Pair)
+	}
+	return resp, nil
+}
+
+// UpdateOrderExecutionLimits sets exchange execution order limits for an asset type
```

Comment:
Looks like you could support the wrapper function `GetFuturesPositionSummary` via https://docs.deribit.com/#private-get_positions